### PR TITLE
[9.0] [PORT] l10n_ch_payment_slip

### PR DESCRIPTION
**Pull request by yvaucher**
- move files in models and view folders
- move description in README.rst
- move financial contributors in README.rst
- update headers as copyright year was missing in most of them

Depends on
- [x] base_transaction_id OCA/bank-statement-reconcile#127

### CODE REVIEW

**Comment by fclementic2c** [comment]
Fonctional test :-1: 

When invoice is validated no BVR is generated

**Comment by philhug** [comment]
@yvaucher please have a look at my fixes:
https://github.com/yvaucher/l10n-switzerland/pull/1

with these changes, I can successfully print a BVR

**Comment by yvaucher** [comment]
Thanks @philhug I merged your changes in this PR

**Comment by fclementic2c** [comment]
3 comments:
- add error message to handle the unvalid v11 file imported
- On the bk St. line : the bvr no must be in field name (like in odoo8) instead of reference
- On the bk St. line : add autocompletion of partner based on bvr no

thx

File: l10n_ch_payment_slip/models/payment_slip.py [hunk 1]
```diff
@@ -165,7 +151,7 @@ def compute_amount(self):
 
     @api.one
```
Comment (yvaucher) [inline l10n_ch_payment_slip/models/payment_slip.py]: Deprecated

**Comment by yvaucher** [comment]
Reference in name field of  bk St. line change was done in `base_transaction_id` module

For the 2 last remarks I suggest to make them in other PRs as enhancements, this module is working already with the same scope as in 8.0.

**Pull request merged**